src/app/features/approver/components/approver-advanced.tsx

```diff
@@ -0,0 +1,58 @@
+import { useRef } from 'react';
+
+import { AnimatePresence, motion } from 'framer-motion';
+import { Flex } from 'leather-styles/jsx';
+
+import type { HasChildren } from '@app/common/has-children';
+import { createDelay } from '@app/common/utils';
+import { Button } from '@app/ui/components/button/button';
+import { Flag } from '@app/ui/components/flag/flag';
+import { ChevronDownIcon } from '@app/ui/icons';
+
+import { useApproverContext } from '../approver.context';
+import { AnimateChangeInHeight } from './animate-height';
+
+const slightPauseForContentEnterAnimation = createDelay(120);
+
+export function ApproverAdvanced({ children }: HasChildren) {
+  const { isDisplayingAdvancedView, setIsDisplayingAdvancedView } = useApproverContext();
```

Comment:
can there be multiple advanced views? if so it seems this will close/open all of them at once?

Reply:
It isn't built to support this, and don't think we will @mica000 ?

Reply:
Maybe I proactively reject two Approver components?

Reply:
Correct, only one advanced view